src/main/java/com/microsoft/azure/functions/worker/handler/WorkerInitRequestHandler.java

```diff
@@ -18,6 +18,12 @@ String execute(WorkerInitRequest request, WorkerInitResponse.Builder response) {
         response.putCapabilities("WorkerStatus", "WorkerStatus");
         response.putCapabilities("RpcHttpBodyOnly", "RpcHttpBodyOnly");
         response.putCapabilities("RpcHttpTriggerMetadataRemoved", "RpcHttpTriggerMetadataRemoved");
+        WorkerMetadata.Builder workerMetadataBuilder = WorkerMetadata.newBuilder();
```

Comment:
Could you please create new function for this code?

Reply:
can you tell me the reason for create a new function for it?

Reply:
To just be more organized, the code used to add the capabilities and now to set the worker metadata. We can have the details of the metadata in other function.